assets/track-events.js

```diff
@@ -67,6 +67,8 @@ var trackEvents = {
     // Clicks on Cloud Platforms in Quick Start Module
     $(".cloud-option").on("click", function() {
       var platformName = $.trim($(this).find(".cloud-option-body").text());
+      //Tracking for SEM Campaign
+      ga("send", "event", "Link", "Click", "Quickstart");
```

Comment:
The static site uses 2 GA tags, and Bruce wants this new event to use the AI tracking code: 117752657. The event that you set up is firing for the other tracking ID.  You'll have to change it to be something like ` ga("newCampaignTracker.send", "event", "Link", "Click", "Quickstart")`. There are some examples in this file. It would also be a good idea to confirm with Bruce that he wants the `eventLabel` for every click to just be "Quickstart." You should also see if he wants to track clicks on the Cloud Partners dropdown and not just the Start Locally module.

Reply:
Actually, the label probably has to be "Quickstart"; otherwise, I think they would need a new goal for each label. But you can still ask about the Cloud Partner clicks just to be sure.

Reply:
Ah ok, I'll update it to fire to the right tracking ID. I did add tracking for the Cloud Partners dropdown on line 96 but I need to update the tracking ID for this one too